pydy/codegen/tests/test_ode_function_generator.py

```diff
@@ -113,6 +115,27 @@ def test_set_linear_system_solver(self):
 
         assert g._solve_linear_system == solver
 
+    def test_no_constants(self):
+        sys = models.multi_mass_spring_damper()
+        sym_rhs = sys.eom_method.rhs()
+        q = sys.coordinates
+        u = sys.speeds
+        p = [sm.Symbol('m0'), sm.Symbol('c0'), sm.Symbol('k0')]
+
+        # Equation sym_rhs with constants substituted with their values
+        sym_rhs2 = sm.MutableDenseMatrix([[sm.Function('v0')(sm.Symbol('t'))],
+                                          [sm.Mul(sm.Pow(1.0, sm.Integer(-1)),
+                                                  sm.Add(sm.Mul(sm.Integer(-1), 2.0,
+                                                                sm.Function('v0')(sm.Symbol('t'))),
+                                                         sm.Mul(sm.Integer(-1),
+                                                                3.0, sm.Function('x0')(sm.Symbol('t')))))]])
+
+        rhs = generate_ode_function(sym_rhs, q, u, p)
+        rhs2 = generate_ode_function(sym_rhs2, q, u, [])
```

Comment:
The fix that I think we should do, should work like so:

```
rhs2 = generate_ode_function(sym_rhs2, q, u)
```

that is, if there are no constants in the system, then `generate_ode_function` shouldn't require them.

Reply:
I shall fix that. I don't seem to understand where the formatting errors are though. I used pylint and flake8 and didn't notice anything out of the ordinary.

Reply:
The formatting errors seem to be fixed.
The command "(flake8 --select W1,W2,W3 --exclude booger . && flake8 --select F401 --exclude booger,__init__.py,conf.py . && flake8 --select W2,W3 --filename='*.js' examples/ pydy/viz/static/js/dyviz/) || (echo "Please run a linter and fix errors, e.g. pylint, flake8, pep8, etc."; false)" exited with 0.
There seems to be some other reason why Travis isn't passing it.

Reply:
It seems to be this : The command "THEANO_FLAGS='gcc.cxxflags="-march=core2"' nosetests -v --with-coverage --cover-package=pydy" exited with 1.